src/components/CardSuits.js

```diff
@@ -0,0 +1,13 @@
+export const suits = {
```

Comment:
we should use naming conventions here,
https://www.syncfusion.com/blogs/post/top-javascript-naming-convention

Reply:
replaced the names of constants